method/group.js

```diff
@@ -114,7 +114,15 @@ module.exports = function GroupMethods (ssb, keystore, state) {
 
       if (!excludeMemberSpec.isValid(content)) return cb(new Error(excludeMemberSpec.isValid.errorsString))
 
-      ssb.publish(content, cb)
+      ssb.publish(content, (err, msg)=> {
+        if (err) return cb(err)
+
+        // TODO: remove the excluded members from the keystore. the reverse of the 
+        //keystore.group.registerAuthors(groupId, authorIds, (err) => {
+        // that invite() does
```

Comment:
I think the only thing that might need to happen is consider undoing `ssb.replicate.request` ... but ... that should also depend on if they're a "friend" outside of this context

I think your work to not rely on keystore for queries is good.

Reply:
hmm good point. but i'm a bit afraid of ssb-friends. or is it that one doing the friend replication? doesn't look like ahau is using replication scheduler? in tribes2 i think both friends+group member replication is delegated to the scheduler, which makes it able to balance these two needs you bring up. should we ignore this problem for now maybe, make a new issue?